share/pkgs/Ubuntu/opennebula.sudoers

```diff
@@ -9,5 +9,8 @@ Cmnd_Alias ONE_OVS = /usr/bin/ovs-ofctl, /usr/bin/ovs-vsctl
 Cmnd_Alias ONE_XEN = /usr/sbin/xentop, /usr/sbin/xl, /usr/sbin/xm
 Cmnd_Alias ONE_CEPH = /usr/bin/rbd
 Cmnd_Alias ONE_HA = /bin/systemctl start opennebula-flow, /bin/systemctl stop opennebula-flow, /bin/systemctl start opennebula-gate, /bin/systemctl stop opennebula-gate, /usr/sbin/service opennebula-flow start, /usr/sbin/service opennebula-flow stop, /usr/sbin/service opennebula-gate start, /usr/sbin/service opennebula-gate stop
+Cmnd_Alias ONE_MOUNT = /bin/mount /var/tmp/*.raw /var/tmp/*, /bin/umount /var/tmp/*
```

Comment:
This should be added to https://github.com/OpenNebula/one/blob/master/share/sudoers/sudo_commands.rb

Also review use of wild char * in sudo

Reply:
Also, all platforms should be refreshed. Ideally, all these commands should be better under a single alias, so they can be easily identified and disabled (e.g., ONE_LXD or LXC).